### PR TITLE
Environment

### PR DESCRIPTION
**Pull request by bmbergh**
##.ENV File
You will need this file which can be sent to you via **Slack** upon request

###Gulpfile
This file is where your dev server will start, your files will be watched and errors will be caught. I created comments in the code but if you have questions please see me. Make you install all the packages for this to work / npm update. 

###Webpack.config file
This file is where your javascript files will by bundled. You will see that the entry point is *'./client/main.jsx'* which where you the entry point for the this React application will begin. 

###Server.js
This file is where I put in the code for sequalizer to work with our database.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    console.log(cyanWatch('Going into dev watch mode...'));
+    setInterval(function() {
+      console.log("Watching...");
+    }, 1000 * 10);
+});
+
+gulp.task('dev', ['jshint','bundle', 'nodemon' ]);
+
+gulp.task('build', ['jshint', 'bundle', 'uglify'])
+
```
Comment (al1271) [inline gulpfile.js]: Lots of white space?

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
+var gulp = require('gulp'),
+	concat = require('gulp-concat'),
```
Comment (al1271) [inline gulpfile.js]: after gulp seems as they don't line up. might be your editor spacing settings

File: gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (al1271) [inline gulpfile.js]: Lots of white space, 82-108

File: gulpfile.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+	sourcemaps = require('gulp-sourcemaps'),
+	gutil = require("gulp-util"),
+	webpack = require("webpack"),
+	WebpackDevServer = require("webpack-dev-server"),
+	webpackConfig = require("./webpack.config.js"),
+	stream = require('webpack-stream'),
+	nodemon = require('gulp-nodemon'),
+	jshint = require('gulp-jshint'),
+	stylish = require('jshint-stylish'),
+	chalk = require('chalk');
```
Comment (al1271) [inline gulpfile.js]: after gulp seems as they don't line up. might be your editor spacing settings

File: .gitignore [hunk 5]
```diff
@@ -29,3 +29,6 @@ node_modules
 
 # Optional npm cache directory
 .npm
+
+# Environment Variables
+.env
```
Comment (icarlosmendez) [inline .gitignore]: This file seems fine except for the fact that it did not prevent the .env file from ending up in the repository.

File: gulpfile.js (hunk 4 above)
Comment (icarlosmendez) [inline gulpfile.js]: Per QA's previous note, this document is using a 4 space tab. Ensure that we adhere to the airbnb style guide by setting it at 2 spaces. Additionally there is an inordinate amount of blank space at the end of the document. Again, with respect to the airbnb style guide, reduce this to one blank line (carriage return) at the end of file. 
I know we are just getting used to this new system so I am accepting of the fact that it has not been corrected. I'm sure we'll find our stride here shortly.

File: webpack.config.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+	      {test: /\.css/, loader: 'style-loader!css-loader'}
+	    ],
+	    noParse: /\.min\.js/
+  	},
+	resolve: {
+    modulesDirectories: ['node_modules'],
+    extensions: ['', '.js', '.json', '.jsx', '.json', '.styl', '.css', '.scss']
+  	},
+  	
+}
```
Comment (icarlosmendez) [inline webpack.config.js]: Per style guide, this file is guilty of the 4 space tab setting.
Also, this file and each file that shows the little red icon on the last line requires one carriage return following the last line of code.

**Comment by reactivepixel** [comment]
This Feature is part of Pull Request #9 and has already been merged.

**Pull request closed without merging**